src/include/ipc/probe.h

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ *         Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+/**
+ * \file include/ipc/probe.h
+ * \brief Probe IPC definitions
+ * \author Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ * \author Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __IPC_PROBE_H__
+#define __IPC_PROBE_H__
+
+#include <ipc/header.h>
+#include <stdint.h>
+
+/* Header for data packets sent via compressed PCM from extraction probes */
+struct probe_data_packet {
+	uint32_t sync_word;
+	uint32_t buffer_id;
+	uint32_t format;
```

Comment:
what is the definition or reference?

Reply:
@plbossart We'll encode sample rate and frame format similarly to cAVS, probably with flag bitmask. To be defined. Exact encoding is not very important at this moment. Will be documented once implemented.

Reply:
@akloniex same as for the sync-word, we do need to have a description for the tools.